lib/models/index.ts

```diff
@@ -51,23 +52,27 @@ export interface User {
 export const FLOWS: Flow[] = [
   { id: 1, name: 'Hello World Flow', definition: [{ type: 'message', message: 'hello' }] },
   {
-      id: 2, name: 'Multiple Choice Flow', definition: [
-          { type: 'message', message: 'Thank you for texting in' },
-          {
-              type: 'multipleChoice',
-              message: 'What is your favorite color?',
-              responses: [
-                  { value: 'red', message: 'You responded "red".', synonyms: [] },
-                  { value: 'green', message: 'You responded "green".', synonyms: [] },
-                  { value: 'blue', message: 'You responded "blue".', synonyms: [] },
-              ]
-          },
-      ]
+    id: 2, name: 'Multiple Choice Flow', definition: [
+      { type: 'message', message: 'Thank you for texting in' },
+      {
+        type: 'multipleChoice',
+        message: 'What is your favorite color?',
+        responses: [
+          { value: 'red', message: 'You responded "red".', synonyms: [] },
+          { value: 'green', message: 'You responded "green".', synonyms: [] },
+          { value: 'blue', message: 'You responded "blue".', synonyms: [] },
+        ]
+      },
+    ]
   },
   {
-      id: 3, name: 'Asking Question Flow', definition: [
-          { type: 'getInfo', message: 'What is your name?', key: 'name' },
-          { type: 'message', message: 'Thank you for sending in your name!' }
-      ]
+    id: 3, name: 'Asking Question Flow', definition: [
+      {
+        type: 'getInfo',
+        message: 'What is your name?',
+        key: 'name',
+        confirmMessage: 'Thank you for sending in your name!'
```

Comment:
It was weird to me that `multipleChoice` had it's submission response baked into the action, but `getInfo` had it as two distinct actions. Seems like `multipleChoice` did this righter, rather than have to index up into the next action for a response.